server/swagger.js

```diff
@@ -52,6 +52,10 @@ const doc = {
       name: 'Security',
       description: 'Endpoints related to security and authentication',
     },
+    {
+      name: 'GPT Assistant Chat',
```

Comment:
The name should be the same as the one as the tag.

Reply:
Completed.